### ISSUE TITLE
DXF files for 3D printing 

### ISSUE THREAD

**Issue by Oceanique** [evanman83/OURS-project#3]
Hello,

I was wondering if you could upload the files you've created in STL ? That's because there's a lot more people who own a 3D printer than a laser cutting machine and I'm one of them ^^. 

Thank you

**Comment by evanman83** [comment]
I didn't design it for 3D printing, but I'm glad you did :) The maximum dimensions for the case (when assembled) are 9cm x 16cm x 3cm, so the main top and bottom pieces should be 90mm x 160mm and all the heights 3mm.

**Comment by evanman83** [comment]
Please consider adding the link to your Thingiverse file into the Readme file under the "Laser cut the phone cover" section, so that everyone can benefit.

**Comment by evanman83** [comment]
Yes those parts are meant to be separable, so that you can pop out the middle panel and access the HDMI port and plug in your monitor instead of the built-in screen if you want. Not NB.
